apipie-rails.gemspec

```diff
@@ -16,7 +16,11 @@ Gem::Specification.new do |s|
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.require_paths = ["lib"]
 
-  s.add_dependency "rails", ">= 4.1"
+  s.add_dependency "actionview"
+  s.add_dependency "actionpack"
```

Comment:
how did you figured `actionpack` and `actionview` was required?
in #741 I am only depending on `railties` and `activemodel`...

Reply:
Actually I am not sure anymore (it's been a bit of time), but:
`railties` actually has `actionpack` as dependency. Thus my dependency should be smaller in this regard, I think.
Not sure why I added `actionview`, thought I'd have search the source code/the commit where the old dependency was replaced by rails.
But I am open to any other solution, as long as they work - everything is better than including `rails` with its raising dependencies. The smaller the footprint, the better it is.